api/src/pcapi/settings.py

```diff
@@ -464,6 +464,7 @@
 BACKOFFICE_SEARCH_SIMILARITY_MINIMAL_SCORE = float(os.environ.get("BACKOFFICE_SEARCH_SIMILARITY_MINIMAL_SCORE", 0.2))
 BACKOFFICE_URL = os.environ.get("BACKOFFICE_URL", "")
 ENABLE_TEST_USER_GENERATION = bool(int(os.environ.get("ENABLE_TEST_USER_GENERATION", 0)))
+ENABLE_USER_DELETION = bool(int(os.environ.get("ENABLE_TEST_USER_GENERATION", 0)))
```

Comment:
[question] Est-ce voulu d'avoir pris la même variable d'environnement que pour `settings.ENABLE_TEST_USER_GENERATION` ?

Si on veut que la même variable d'environnement contrôle ces 2 settings, on pourrait simplement écrire : 

```suggestion
ENABLE_USER_DELETION = ENABLE_TEST_USER_GENERATION
```

Je pense que ce serait plus explicite et on ne se poserait pas de question sur un éventuel copier-coller raté. :)